### ISSUE TITLE
PlaceholderText not showing

### ISSUE THREAD

**Issue by Mithicor** [lepoco/wpfui#1187]
### Describe the bug

It seems that beyond 3.0.4 the PlaceholderText no longer shows for me.

### To Reproduce

default behavior.

### Expected behavior

the place holder text should be visible when the textbox is empty

### OS version

OS: Windows 11 Enterprise N
Version: 23H2

### .NET version

4.8.1

### WPF-UI NuGet version

3.0.5 or later.

**Comment by textGamex** [comment]
Please provide more information.

**Comment by Mithicor** [comment]
Odd... I just tried updating the Version to 3.0.5 again and it seems to be working now.

I am not sure what was going on, but when I submitted the bug 2 weeks ago, I switched back and forth between versions 3.0.5 and 3.0.4 several times to verify the bug. Whenever I went to version 3.0.5 the 'PlaceholderText' wouldn't display. It was just showing an empty Textbox. I used just the default Textbox shown below to test it.

`<ui:TextBox PlaceholderText="This is a Test" Text=""/>`


If it starts happening again I will update this with any new information.
